sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/postgres/PostgresBoolTests.java

```diff
@@ -1,11 +1,33 @@
 package org.dbsp.sqlCompiler.compiler.postgres;
 
+import org.dbsp.sqlCompiler.compiler.DBSPCompiler;
 import org.junit.Test;
 
 /**
  * <a href="https://github.com/postgres/postgres/blob/master/src/test/regress/expected/boolean.out">boolean.out</a>
  */
 public class PostgresBoolTests extends PostgresBaseTest {
+    @Override
+    public void prepareData(DBSPCompiler compiler) {
+        compiler.compileStatements(
+                        // "CREATE TABLE BOOLTBL1 (f1 bool);\n" +
+                        // "INSERT INTO BOOLTBL1 (f1) VALUES (bool 't');\n" +
+                        // "INSERT INTO BOOLTBL1 (f1) VALUES (bool 'True');\n" +
+                        // "INSERT INTO BOOLTBL1 (f1) VALUES (bool 'true');\n" +
```

Comment:
Are these cases that don't correctly work right now?

Reply:
These indeed don't work, Calcite does not accept this syntax for Boolean literals.
I have also not included the tests that depend on these tables yet.